www/src/components/Menu.jsx

```diff
@@ -0,0 +1,194 @@
+// are for page creation purposes.
+const menuQuery = graphql`
+  query menuQuery {
+    components: allMdx(
+      filter: {
+        parent: {
+          internal: { owner: { nin: "gatsby-transformer-react-docgen" } }
+        }
+        frontmatter: { type: {} }
+      }
+      sort: { fields: frontmatter___title }
+    ) {
+      categories: group(field: frontmatter___categories) {
+        nodes {
+          ...ComponentPage
+        }
+        fieldValue
+      }
+      types: group(field: frontmatter___type) {
+        nodes {
+          ...ComponentPage
+        }
+        fieldValue
+      }
+      all: nodes {
+        ...ComponentPage
+      }
+    }
+  }
+
+  fragment ComponentPage on Mdx {
+    id
+    frontmatter {
+      categories
+      type
+      title
+    }
+    fields {
+      slug
+    }
+  }
+`;
+
+const ComponentNavItem = ({
+  id, fields, frontmatter, ...props
+}) => (
+  <li {...props}>
+    <Link to={fields.slug}>{frontmatter.title}</Link>
+  </li>
+);
+
+ComponentNavItem.propTypes = {
+  id: PropTypes.string.isRequired,
+  fields: PropTypes.shape({
+    slug: PropTypes.string.isRequired,
+  }).isRequired,
+  frontmatter: PropTypes.shape({
+    title: PropTypes.string.isRequired,
+  }).isRequired,
+};
+
+const MenuComponentList = ({ children }) => (
+  <div className="menu-component-list">{children}</div>
+);
+
+MenuComponentList.propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+const MenuComponentListCategory = ({ children, title }) => (
+  <div className="menu-component-list-category">
+    <h5>{title}</h5>
+    {children}
+  </div>
+);
+
+MenuComponentListCategory.propTypes = {
+  children: PropTypes.node.isRequired,
+  title: PropTypes.string.isRequired,
+};
+
+const Menu = () => {
+  const { components } = useStaticQuery(menuQuery);
+  const { categories, all } = components;
+
+  return (
+    <Container size="xl" className="py-5">
+      <Row className="mb-5">
+        <Col sm={3}>
+          <h3 className="mb-3">Guides</h3>
+          <ul className="list-unstyled small mb-4">
+            <li>
+              <Link to="/guides/installation-and-usage">
+                Installation and usage
+              </Link>
+            </li>
+            <li>
+              <a
+                target="_blank"
+                rel="noopener noreferrer"
+                href="https://openedx.atlassian.net/wiki/spaces/BPL/pages/1773502564/Component+Contribution+Process"
+              >
+                Contributing components
+              </a>
+            </li>
+          </ul>
+          <h3 className="mb-3">Resources</h3>
+          <ul className="list-unstyled small mb-4">
+            <li>
+              <Link to="/status">
+                Component status
+              </Link>
+            </li>
+            <li>
+              <Link to="/insights">
+                Usage insights
+              </Link>
+            </li>
+            <li>
+              <a
```

Comment:
Huh, when I ran the docs site locally, this <a> tag is a button, and it's non-functional.